index.js

```diff
@@ -503,11 +507,8 @@ exports.buildModule = function (funcs, imports = [], exports = [], cbs = []) {
   (memory 500)
   (export "memory" (memory 0))
 
-  (table
-    (export "callback")
-    anyfunc
-    (elem ${cbs.join(' ')})
-  )
-   ${funcStr}
+  ${callbackTableStr}
```

Comment:
This changes the callbacks from being a table into exported functions. These are two different things. (Not saying the old version worked properly.)

Reply:
Ah, thanks. ewasm-kernel didn't expect a table, I should've fixed it there originally.